.github/bin/s3/delete-s3-bucket.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+S3_SCRIPTS_DIR="${BASH_SOURCE%/*}"
+
+if [[ ! -f "${S3_SCRIPTS_DIR}/.bucket-identifier" ]];  then
+    echo "Missing file ${S3_SCRIPTS_DIR}/.bucket-identifier"
+    exit 1
+fi
+
+S3_BUCKET_IDENTIFIER=$(cat "${S3_SCRIPTS_DIR}"/.bucket-identifier)
+
+echo "Deleting AWS S3 bucket ${S3_BUCKET_IDENTIFIER}"
+aws s3api delete-bucket \
+  --bucket "${S3_BUCKET_IDENTIFIER}" \
+  --region "${AWS_REGION}"
+
+echo "Waiting for AWS S3 bucket ${S3_BUCKET_IDENTIFIER} to be deleted"
+
+aws s3api wait bucket-not-exists \
```

Comment:
What's the timeout on this? We need to make sure not to block the CI indefinitely.

Reply:
Thinking about it now, waiting for the bucket to not exist is a total waste here. We aren't going to do anything if it doesn't get cleaned up by this step, so why even check?

Reply:
Good point, but my question also applies to the wait in the creation script.

Reply:
From the s3api docs:

https://awscli.amazonaws.com/v2/documentation/api/2.0.34/reference/s3api/wait/bucket-exists.html
> Wait until 200 response is received when polling with head-bucket. It will poll every 5 seconds until a successful state has been reached. This will exit with a return code of 255 after 20 failed checks.

So it looks like a max of 100 seconds